environments/demo/backend_lb_config.yaml

```
@@ -314,3 +312,7 @@ gateways:
     # Log and Audit
       - product: lau
         component: idam-backend
+
+    # AAC (CDM)
+      - product: aac
+        component: aac-ac-int-manage-case-assignment
```

you shouldn't be doubling that up unless you want the url to be 
`aac-aac-ac-int-manage-case-assignment.core-compute-...`
```suggestion
        component: ac-int-manage-case-assignment
```